src/function/gds/gds_utils.cpp

```diff
@@ -19,6 +21,19 @@ GDSComputeState::GDSComputeState(std::unique_ptr<function::FrontierPair> frontie
 
 GDSComputeState::~GDSComputeState() = default;
 
+void GDSUtils::initKCore(processor::ExecutionContext* context, RJCompState& rjCompState, graph::Graph* graph, std::unique_ptr<EdgeCompute> initEdgeCompute) {
```

Comment:
Remove. GDSUtils should not hold any algorithm specific logic.

Reply:
Gotcha, found a way to use runUntilConvergence to do the calculation instead